### PR TITLE
Create GreenLake “data-services-on-the-hpe-greenlake-platform/home”

### PR DESCRIPTION
**Pull request by rondharma**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (choukden) [inline content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:15]: Hi Ron,

In the preview https://deploy-preview-2531--hpe-dev-portal.netlify.app/greenlake/data-services-on-the-hpe-greenlake-platform/home/ , the font for the numbered item lists is different than the rest of the content. 

Can you please try to insert the following style statement at the top of the page instead of the other two style statement to see whether the rendering would be better? 

Note: The fonts may appear bigger than the standard text in the right panel of the CMS editor. This is expected. However, the Netlify process should render the text correctly (as standard text size) when creating the “Preview” of the post and when publishing the post to the HPE Developer Community portal.


<style>
li {
   font-size: 27px;
   line-height: 33px;
   max-width: none;
}
</style>

 instead of the two styles:

<style>
ul li{
 font-size:27px;
}
</style>

<style>
ol li{
 font-size:27px;
}
</style>

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (choukden) [inline content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:27]: Naming convention for HPE GreenLake edge to cloud platform is: HPE Greenlake edge-to-cloud platform.

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (choukden) [inline content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:38]: HPE GreenLake edge-to-cloud platform (or simply use HPE GreenLake platform).

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (choukden) [inline content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:44]: after the release of the HPE GreenLake edge-to-cloud platform, Data Services introduce....

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (choukden) [inline content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:48]: on the HPE GreenLake edge-to-cloud platform in **March 2024**, ...

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (choukden) [inline content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:61]: Second sentence, add "HPE" in front of GreenLake documentation about the APIs...

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (choukden) [inline content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:73]: In this paragraph, occurrences of "Open API" should be spelled "OpenAPI"

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (Didier-Lalli) [reply content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:15]: You should also add in the same style section:    line-height: 33px;
To give a little more room between lines.

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (Didier-Lalli) [reply content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:27]: We need to make sure we follow the latest name rules for HP Greenlake. It's changing AGAIN

**Review by Didier-Lalli — CHANGES_REQUESTED** [review]

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (choukden) [reply content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:27]: @rondharma 
We recently received the information below for Discover (From Kailah Findensen):

--> Request to move forward is to drop “edge-to-cloud platform” or “platform” in any reference to GreenLake. So at this point we should only be saying “HPE GreenLake”

However, Paul Zinn shared the following information:
this guidance is early, as the full decision on how to refer to HPE GreenLake and the platform hasn’t been made yet. I would recommend holding off on implementing Kailah’s ask for now so that you don’t end up making two passes through your material instead of one. When I hear of the final decision, I’ll pass it on.
 

So, until further notice, let's stick with the current naming convention:
HPE GreenLake edge-to-cloud platform or HPE GreenLake platform

Sorry for any confusion this might create.

Thanks
Denis

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (rondharma) [reply content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md]: Got it.. I have applied the single style as suggested by Denis which already included Didier suggestion of the line-height: 33px

File: content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md
Comment (rondharma) [reply content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md:21]: Thanks Denis, Didier

**Pull request merged**
